pipeline-scripts/release.groovy

```diff
@@ -21,24 +23,64 @@ def stageValidation(quay_url, name, advisory) {
         error("Payload ${name} already exists! Cannot continue.")
     }
 
-    // AMH - This may be optional?
-    if (advisory != "") {
-        echo "Verifying advisory exists"
+    if (!advisory) {
+        echo "Getting current advisory for OCP $version from build data..."
         res = commonlib.shell(
                 returnAll: true,
-                script: "elliott get ${advisory}"
-        )
+                script: "elliott --group=openshift-${version} get --json - --use-default-advisory image",
```

Comment:
even though `commonlib.shell` is used under the hood, i've noticed that in other jobs `buildlib.elliott` is used whenever we want to perform `elliott` commands.
https://github.com/openshift/aos-cd-jobs/blob/master/pipeline-scripts/buildlib.groovy#L107-L111
usage example: https://github.com/openshift/aos-cd-jobs/blob/master/jobs/build/sweep/Jenkinsfile#L45

 don't know the reasons for that, but would recommend that you make use of it as well, for consistency.

Reply:
I haven't realized that method. `commonlib.shell()` is frequently used in this file so  I just copied it. I think we should make a better use of `buildlib.elliott`.